mappings/net/minecraft/block/AbstractBlock.mapping

```diff
@@ -535,10 +540,17 @@ CLASS net/minecraft/class_4970 net/minecraft/block/AbstractBlock
 			FIELD field_16560 extrudedFaces [Lnet/minecraft/class_265;
 			FIELD field_17651 exceedsCube Z
 			FIELD field_19360 collisionShape Lnet/minecraft/class_265;
-			FIELD field_19429 solidFullSquare [Z
+			FIELD field_19429 solidSides [Z
 			FIELD field_20337 isFullCube Z
+			FIELD field_25830 SHAPE_TYPE_LENGTH I
```

Comment:
weirdly named, SHAPE_TYPES_LENGTH, or SHAPE_TYPE_COUNT probably better. but since it's private -> largely irrelevant

Reply:
it's the number of possible shape types, used to flatten 2d index to an 1d array index